### PR TITLE
qm-create: add German translation

### PR DESCRIPTION
**Pull request by Braintelligence**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
version 7.2, Wed 04 May 2022 07:30:00 AM CEST

### CODE REVIEW

File: pages.de/linux/qm-create.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Lege den Typ des Betriebssystems auf der virtuellen Maschine fest:
+
+`qm create {{100}} --ostype {{win10}}`
+
+- Ersetze eine bestehende virtuelle Maschine (setzt deren Archivierung voraus):
+
+`qm create {{100}} --archive {{path/to/backup_file.tar}} --force 1`
+
+- Lege ein Skript fest, welches automatisch abhängig vom Zustand der virtuellen Maschine ausgelöst werden soll:
```
Comment (kbdharun) [inline pages.de/linux/qm-create.md:22]: Can you rephrase this a bit to better align with our main page? 

> I checked this with Google translate yours read ` Set a script to be triggered automatically depending on the state of the virtual machine:` but the main page has `Specify a script that is executed on specific triggers during machine lifetime:`

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr, Thanks for your contribution. I have a suggestion for this page.

File: pages.de/linux/qm-create.md (hunk 1 above)
Comment (Braintelligence) [reply pages.de/linux/qm-create.md:22]: The main page is somewhat incorrect here. I already talked to the author, he will propose a correction in the near future.

See https://pve.proxmox.com/pve-docs/qm.1.html where it says:
```
It will be called during various phases of the guests lifetime. For an example and documentation see the example script under /usr/share/pve-docs/examples/guest-example-hookscript.pl.
```

The script in question: https://git.proxmox.com/?p=pve-docs.git;a=blob;f=examples/guest-example-hookscript.pl;hb=HEAD

You can see this is all about pre-start/stop and post-start/stop states of the virtual machine. There is no other "specific trigger", it's all about the states.

I will translate the paths.

File: pages.de/linux/qm-create.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# qm create
+
+> Erzeugung einer virtuellen Maschine per QEMU/KVM Virtual Machine Manager.
+> Weitere Informationen (Englisch): <https://pve.proxmox.com/pve-docs/qm.1.html>.
```
Comment (kbdharun) [inline pages.de/linux/qm-create.md:4]:
```suggestion
> Weitere Informationen: <https://pve.proxmox.com/pve-docs/qm.1.html>.
```

File: pages.de/linux/qm-create.md (hunk 1 above)
Comment (Braintelligence) [reply pages.de/linux/qm-create.md:22]: Since the main page author already provided a PR for correction and I also translated the paths I will mark this as resolved.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]
